tests/unit/h/accounts/schemas_test.py

```diff
@@ -194,18 +191,6 @@ def test_it_is_valid_if_email_same_as_users_existing_email(
 
         schema.deserialize({"email": user.email, "password": "flibble"})
 
-    def test_it_is_invalid_if_csrf_token_missing(self, pyramid_request, schema):
-        del pyramid_request.headers["X-CSRF-Token"]
-
-        with pytest.raises(BadCSRFToken):
-            schema.deserialize({"email": "foo@bar.com", "password": "flibble"})
-
-    def test_it_is_invalid_if_csrf_token_wrong(self, pyramid_request, schema):
-        pyramid_request.headers["X-CSRF-Token"] = "WRONG"
-
-        with pytest.raises(BadCSRFToken):
-            schema.deserialize({"email": "foo@bar.com", "password": "flibble"})
```

Comment:
The Colander schemas no longer do CSRF verification. Several unittests like this can now be removed. (In most cases developers had forgotten to add CSRF test cases for their schemas anyway.)